content/about.md

```diff
@@ -1,12 +1,10 @@
 ---
 title: "About me"
-template: "page"
-socialImage: "/media/image-2.jpg"
```

Comment:
`socialImage` is an important metadata for search engines, you can't remove the image. Either change the image to something more relevant or keep the old one. In any case, you need to keep the `socialImage`.